casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -14,26 +14,27 @@ import scala.concurrent.SyncVar
 import scala.util.{Failure, Success, Try}
 
 //runtime is a SyncVar for thread-safety, as all checkpoints share the same "hot store"
-class Checkpoint private (val hash: ByteString, runtime: SyncVar[Runtime]) {
+class RuntimeManager private (runtime: SyncVar[Runtime]) {
+  type StateHash = ByteString
```

Comment:
I'd prefer if we didn't give the same type alias twice. Perhaps use `import RuntimeManager.StateHash` here.